content/post/0-to-60-analytics-stack.md

```diff
@@ -0,0 +1,79 @@
+
+Here is a simple chart I wish I would have used to diagnose analytics needs:
+
+![Analyst or Self Service](https://nelsonauner.com/images/analysts_or_self_serve.png)
+
+
+This concludes the three main “courses” of an analytics stack. If you are still hungry, I will touch briefly on two increasingly necessary components as you continue to grow:
+
+## Business Logic Layer:  
+- You might consider: Looker, DBT
+
+ Once you get to more than 3 SQL writers, each will use slightly different JOIN conditions and aggregations, and it will cause problems. These problems may be technical (the same expensive queries getting run repeatedly) but certainly analytical problems as different teams build out slightly different queries for the same metrics.
+Business logic layers solve this problem by centralizing and documenting the (business logic <-> data) interface.
+
+I was comparing DBT with Looker, but the important takeaway is to consider the tradeoffs between a comprehensive system  DBT overlaps a lot with LookML in this regard. If you don’t use Looker, you should definitely use DBT. If you do use Looker, the gains to using DBT are less. I opted to use DBT because it will make it much easier to move off of Looker, if I ever need to, allows us to use DBT’s cleaned data for data science instead of trapping that data in Looker, and has some more advanced features not found in LookML's PDTs. You can opt for DBT open source or their paid hosted version, DBT Cloud. DBT Cloud is more user friendly, but still requires a fair amount of technical knowledge to operate.
+
+Org Discussion Points: Since DBT is either free (run yourself) or very cheap (dbt cloud), this is likely an easy talk. I regret setting up DBT on an EC2 box instead of going directly to dbt’s paid cloud offering. DBT is much more technical than looker, so your average business user is either going to need to level up their command line skills, or not be involved in the DBT process. DBT Cloud doesn't offer automated user provisioning, so this will be yet-another-tool to admin. Se la vie!
+
+## Org & On-going training:
+
+Once you reach about 100 employees, the way you communicate cross-org is going to be more important than everything I just wrote about (unless you really screwed it up). There is a great post on this by TJ Murphy: [Lessons learned managing the GitLab Data team](https://about.gitlab.com/blog/2020/02/10/lessons-learned-as-data-team-manager/). For a deeper dive on org structure (embedded or centralized analysts?), selecting KPIs, and driving prioritization, I recommend Carl Anderson’s “Creating a Data-Driven Organization” and, of course, [Locally Optimistic blog](https://www.locallyoptimistic.com/)
```

Comment:
This section feels a bit tacked on. I'd either flesh out what you mean by "on-going training" and "the way you communicate cross-org" and why they are so important, or change the heading to something like "Thinking Ahead" to capture the fact that the tools are just the beginning.